manual/tools/Makefile

```diff
@@ -1,8 +1,7 @@
-TOPDIR=../..
-COMPFLAGS=-I $(OTOPDIR)/otherlibs/str -I $(OTOPDIR)/otherlibs/unix
-include $(TOPDIR)/Makefile.tools
+uTOPDIR=../..  COMPFLAGS=-I $(OTOPDIR)/otherlibs/str -I
+$(OTOPDIR)/otherlibs/unix include $(TOPDIR)/Makefile.tools
```

Comment:
Fat fingers? Your text editor has re-wrapped these three lines into two and you have an extraneous `u` at the beginning of the first one. This probably explains the Appveyor failure that we saw earlier today.